omero/users/cli/containers-annotations.txt

```diff
@@ -0,0 +1,63 @@
+Creating containers and annotations
+-----------------------------------
+
+The :omerocmd:`obj` command allows users to create and update OMERO objects,
```

Comment:
Not quite comfortable with the ", this" break here, it feels like it needs something more substantial?